SPECS/liblogging/liblogging.spec

```diff
@@ -0,0 +1,78 @@
+Summary:        Logging Libraries
+Name:           liblogging
```

Comment:
Please also add an entry in `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md `. I believe we have all of our packages mentioned there.

Reply:
Thanks- fixes this comments and the others in this pass.